drone/src/plan.rs

```diff
@@ -48,12 +48,22 @@ impl DronePlan {
         };
 
         let proxy_options = if let Some(proxy_config) = config.proxy {
+            let (bind_port, bind_redir_port, key_pair) = if let Some((key_pair, https_port)) =
```

Comment:
The zip makes this hard to read, I'd prefer

    if let (Some(key_pair), Some(https_port)) = (config.cert, proxy_config.https_port)